### PR TITLE
Adding DBnomics

### PR DESCRIPTION
**Pull request by michaelweinold**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by michaelweinold** [comment]
@cbenz please add a comment in this thread confirming you want to be added as a feedstock maintainer (compare the PR checklist at the top).

**Comment by michaelweinold** [comment]
I confirm I want to be added as a maintainer.

**Comment by dopplershift** [comment]
Please don't modify recipes outside your own. Only add your recipe.

**Comment by michaelweinold** [comment]
> Please don't modify recipes outside your own. Only add your recipe.

Whoops, sorry. I thought I had eroneously pulled these files into my branch with https://github.com/conda-forge/staged-recipes/pull/21750/commits/4583ba867280adee7f5f1e823f823b684624c14a. Commit has been reverted.

File: recipes/dbnomics/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - dbnomics
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://git.nomics.world/dbnomics/dbnomics-python-client
+  summary: DBnomics Python Client
+  license: AGPL-3.0-only
```
Comment (dopplershift) [inline recipes/dbnomics/meta.yaml:39]: Looks like it's actually ["or later"](https://git.nomics.world/dbnomics/dbnomics-python-client/-/blob/master/dbnomics/__init__.py#L10):
```suggestion
  license: AGPL-3.0-or-later
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Thanks for cleaning up the commit. Just a minor tweak to the metadata and this looks good.

I also need confirmation from @cbenz that they are willing to be listed as a maintainer for this recipe.

**Comment by cbenz** [comment]
I confirm I want to be added as a maintainer.

**Comment by michaelweinold** [comment]
@dopplershift - I believe this PR is now ready for review. Thank you for your patience!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**